packages/angular/cli/commands/update.json

```diff
@@ -63,8 +63,8 @@
           "type": "boolean",
           "default": false
         },
-        "skipCommits": {
-          "description": "Do not create source control commits for updates and migrations.",
+        "createCommits": {
```

Comment:
I would prefer `commitUpdates` or `commitChanges` as I think that is more descriptive since it includes a hint at what is in the commits.

I guess the current version does hint at the fact that it might create more than a single commit.

However, as a user, I feel like 'createCommits' is more vague and will confuse many developers.